### ISSUE TITLE
Same icons on the screenshot

### ISSUE THREAD

**Issue by duard** [horst3180/arc-icon-theme#28]
How to install this icons used on this screenshot ? 
https://cloud.githubusercontent.com/assets/10381709/17079474/b1a91c6a-5126-11e6-8e05-e3b9212e17d1.png

**Comment by gsag** [comment]
Found on readme. Look for the last line - https://github.com/horst3180/arc-theme
Screenshot Details: 
Icons: Arc 
Launcher Icons based on White Pixel Icons
Font: Futura Bk bt

**Comment by duard** [comment]
Yes, but how can I config my Dock Bar to use this icons White Pixel ?

**Comment by gsag** [comment]
Hmm... Probably look for the Dock theme and change some configuration ... https://github.com/horst3180/arc-theme/blob/master/extra/Arc-Plank/dock.theme
Sorry, but I don't know such tecnical details...

**Comment by duard** [comment]
Olhei lá guilherme, e não vi nada apontando para os icones daquele pacote

**Comment by gsag** [comment]
Provavelmente não tenha a configuração no arquivo, mas ela deve existir. Já tentou dar uma olhada na internet pelas configurações do Plank?

**Comment by duard** [comment]
A única coisa que quero é os icones na Dash Dock serem iguais aos do screenshot oficial do theme ... já pesquisei tudo, não encontro como fazer. 

![screenshot from 2016-10-15 17-11-27](https://cloud.githubusercontent.com/assets/36894/19413086/95ee4e76-92fa-11e6-8946-cb275b042555.png)

**Comment by duard** [comment]
Não sabia dessa dock chamada Plank, instalei aqui, vamos ver se por ela consigo alguma melhora no visual. A Dash Dock é muito limitada.

**Comment by duard** [comment]
Usei as configurações do PLank mas os icones não mudam, continuam icones normais.
Falta somente os icones White Pixel para ficar legal.

![screenshot from 2016-10-15 17-25-39](https://cloud.githubusercontent.com/assets/36894/19413167/70c212de-92fc-11e6-9efa-65fd8d7ecd01.png)

**Comment by gsag** [comment]
Encontrei isso: http://www.noobslab.com/2016/03/latest-plank-dock-and-plank-themer.html
Dê uma olhada.

**Comment by duard** [comment]
No Plank consigo criar .desktop com o icone que quero e ao executar ele aparece na Plank Dock e posso fixa-lo lá ... mas o theme oficial usa a DashDock e mesmo alterando nos .desktop em /usr/share/applications ~/.local/share/applications os icones continuam aparecendo como se fossem do ubuntu mesmo.

**Comment by duard** [comment]
Deu certo cara, reiniciei para matar a dúvida.
Um saco ter que reiniciar.

![screenshot from 2016-10-15 18-17-53](https://cloud.githubusercontent.com/assets/36894/19413448/b7c7f0ac-9303-11e6-9ec9-ad0015a7bc8e.png)

**Comment by gsag** [comment]
Legal! Se o seu problema foi resolvido, só fechar esta issue. 
Valeu!

**Comment by why7e** [comment]
Sorry for the bump, I don't speak Portugese, how did you get it to work?

**Comment by duard** [comment]
@Sir-Lagsalot you will need to edit/create all app icons :-/ ... much work
